presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/AssessmentFragment.kt

```diff
@@ -70,6 +70,13 @@ open class AssessmentFragment : Fragment() {
                     this.showStep(showNodeState)
                 }
             })
+        start()
+    }
+
+    /**
+     * Starts the Assessment by calling start on the [viewModel]. This is called in onActivityCreated.
+     */
+    open fun start() {
```

Comment:
Wondering if this should follow the pattern we used elsewhere of:

```

open fun willStart() {
}

open fun didStart() {
}

// and then above:
willStart()
viewModel.start()
didStart()
```

Or is this intended to allow them to delay the start until they have asynchronously loaded the database?

Reply:
This is to allow them to delay the start until they have asynchronously initialized their engine.